### DISCUSSION TITLE
How can I disable telescope as depency?

### DISCUSSION THREAD

**Discussion by niamleeson** [rmagatti/auto-session#292] [Q&A] [answered]
I don't use telescope and telescope dependency is causing 130ms+ slowdown on startup. I'd like to cut it as much as possible. How can I remove/disable telescope dependency?

**Comment by rmagatti** [comment]
Is anything happening when you just don't define Telescope as a dependency?

**Comment by niamleeson** [comment, marked as answer]
Added this in setup options and this disabled telescope for me

```
session_lens = {
  load_on_setup = false,
},
```
